src/task_scl_eff_pot_hab.py

```diff
@@ -60,10 +55,10 @@ class SCLPolygons(SCLTask):
         },
     }
     thresholds = {
-        "str_lc": 50,
+        "str_lc": 4,
```

Comment:
I think this var name should be a bit less abbreviated; it wasn't clear to me what it was

Reply:
renamed to structural_habitat